changelog/7348.feature.rst

```diff
@@ -0,0 +1 @@
+Added an example of plotting a rectangle on a map with a rotation angle relative to the axes using :meth:`~sunpy.map.GenericMap.draw_quadrangle` using ``SkyOffsetFrame``
```

Comment:
```suggestion
Add an example of plotting a rectangle on a map with a rotation angle relative to the axes.
```
Can you also add the sphinx interlink to the gallery example as well?

Reply:
could you elaborate this please?

Reply:
Hopefully https://sphinx-gallery.github.io/stable/auto_examples/plot_0_sin.html#cross-referencing helps